twilio/http/validation_client.py

```diff
@@ -93,3 +95,14 @@ def _get_host(self, request):
         """Pull the Host out of the request"""
         parsed = urlparse(request.url)
         return parsed.netloc
+
+    def validate_ssl_certificate():
+        """
+        Validate that a request to the new SSL certificate is successful
+        :return: null on success, raise TwilioRestException if the request fails
+        """
+        client = TwilioHttpClient()
+        try:
+            response = client.request('GET', 'https://api.twilio.com:8443')
```

Comment:
I think we need to check the status code on the response.

Also, @imthepitts's idea was that there should be a way of getting at the request and response object if it fails, and `TwilioHttpClient`'s `last_request`/`last_response` members seem like a decent way of achieving that.

I would propose accepting a `TwilioHttpClient` as an argument to this method, and raising a `TwilioRestException` if it has a non-success response code. I wouldn't bother trying to catch other exceptions -- it seems like it will be easier to debug problems if we let them bubble up to the client rather than swallowing them and returning this bogus 500 instead.